added my contribution

AdeyefAbiola opened the pull request.

acekyd commented:
There's a conflict here, you edited my file so there's a conflict @AdeyefAbiola. To fix this, just look at the contributions/acekyd.html file

Commit:
added my contribution

Commit:
initial commit

Commit:
change 2

AdeyefAbiola committed:
Delete read.txt

AdeyefAbiola committed:
Add generic family to the font-family css property

 adding a generic family since there's no guarantee that a specific font is installed on the computer or can be downloaded using a @font-face at-rule. Also i add an additional fonts so that the browser can have a fall back of font if one is not available.

AdeyefAbiola committed:
Adding generic family to the font-family Property

adding a generic family since there's no guarantee that a specific font is installed on the computer or can be downloaded using a @font-face at-rule. Also add an extra font-type for the browser the users is using to have a fall back in case the font-type is not present, and if none of the two isn't present, the browser can pick from any of the 'sans-serif' generic family and not serif, cursive, monospace and fantasy family.
